geoapps/octree_creation/constants.py

```diff
@@ -99,6 +99,12 @@
             "main": True,
             "value": 500.0,
         },
+        "diagonal_balance": {
+            "group": "Basic",
+            "label": "UBC Compatible",
+            "main": True,
+            "value": False,
```

Comment:
```suggestion
            "value": True,
```

I think we can default to True. You might have to turn it off for the unittests though to avoid changing the inversion runs.